developer-docs-site/docs/guides/building-from-source.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Building Aptos From Source"
+slug: "build-from-source"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Building Aptos From Source
+
+[Binary releases are available](../cli-tools/aptos-cli-tool/index.md), but if you want to build from source or develop on the Aptos tools, this is how.
+
+## Supported operating systems
+
+Aptos can be built on various operating systems, including Linux, macOS. and Windows. Aptos is tested extensively on Linux and macOS, and less so on Windows. Here are the versions we use:
+
+* Linux - Ubuntu version 20.04 and 22.04
```

Comment:
Don't forget Arch Linux :grimacing: 

While it is tested and somewhat maintained on there for some purposes, it likely doesn't belong here.

Reply:
Whatever becomes true in the future, it'll be easier to update in this one place! ;-)